Add bulk redirects to Pages how to section. (#3454)
cloudflare/cloudflare-docs: products/pages/src/content/how-to/www-redirect.md

In this tutorial, you will learn how to redirect the `www` subdomain to your apex domain (`example.com`). This is a common question and can be achieved by using [Page Rules](https://support.cloudflare.com/hc/en-us/articles/218411427).

<Aside type= "note">

> It is recommended to use [Bulk Redirects](https://developers.cloudflare.com/pages/how-to/use-bulk-redirects.md) instead of Page Rules for redirecting domains. However, using Pages Rules will work as expected.

Reviewer: This sentence confuses me a little. Are we saying this article (using page rules) is the right way or the wrong way? 

If it's right, this maybe should say "Although generally recommended to use [Bulk Redirects] when redirecting between domains, in the simple case where you just want a single redirect from `www.example.com` to `example.com`, a single Page Rule will work just as well." or something.

If it's wrong, it feels there should be an equivalent how-to for www->apex but using bulk redirects

Author: I think you are right, bulk redirects is a Page rule but recommended for more than our pages _redirects can handle at this time. I will update

> <Aside type= "note">
> Although generally recommended to use [Bulk Redirects](./use-bulk-redirects.md) when redirecting between domains, in the simple case where you just want a single redirect from www.example.com to example.com, a single Page Rule will work just as well.
> </Aside>

Reviewer 2: ```suggestion
<Aside type= "note">

Although it is generally recommended to use [Bulk Redirects](./use-bulk-redirects.md) when redirecting between domains, for smaller use cases, such as a single redirect from `www.example.com` to `example.com`, a single Page Rule will work just as well.

</Aside>
```
